index.html

```diff
@@ -4,9 +4,124 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./node_modules/bootstrap/dist/css/bootstrap.min.css">
```

Comment:
- [ ] This method of linking bootstrap is wrong. Since the project has no package.json therefore there is no node_modules. So to link bootstrap correctly kindly follow the instructions in this [link](https://getbootstrap.com/docs/4.3/getting-started/introduction/).